uber/site_sections/registration.py

```diff
@@ -1,18 +1,10 @@
 from uber.common import *
 
-def check_everything(attendee):
```

Comment:
In case it wasn't clear, we're no longer doing ``MODE`` checks; the whole point of the current approach is to have anything that needs be customized able to be customized by a new API.  (There are probably some customizations we haven't needed to do yet but will need to do; in those cases we just need to add in some more extensibility rather than putting a ton of if-statements like this into our code.)